CHANGELOG.md

```diff
@@ -33,6 +33,162 @@ All sections are explained on the link above, they are all optional, and each of
 
 <!-- New entries go after this line -->
 
+## v2.5.0 - 2017-10-@@@@@@
+
+For more details, [see the full changelog](https://github.com/thelounge/lounge/compare/v2.4.0...v2.5.0) and [milestone](https://github.com/thelounge/lounge/milestone/22?closed=1).
+
+Our [v2.3.0 release](https://github.com/thelounge/lounge/releases/tag/v2.3.0) was big, well, v2.5.0 is even bigger! 🎉
+
+If you are a client user, you will first notice that The Lounge is now using your OS system font, then that status messages (joins, parts, nick changes, etc.) are now condensed with each other.
+
+<p align="center">
+  <img width="555" alt="Condensed status messages displaying 3 users have joined the channel, 4 users have quit, and 1 user has changed nick" src="https://user-images.githubusercontent.com/113730/31530599-a6d719f2-afaf-11e7-9b82-db0365c41c4e.png">
+  <br>
+  <em>Condensed status messages</em>
+</p>
+
+After a while, you'll realize that the client now automatically reconnects when losing network (farewell, `Client connection lost — Click here to reconnect`!), and that channel backlog now automatically loads when scrolling. Unfortunately, that last bit forced us to phase out the `/clear` command for technical reasons.
+
+If you go to the _Settings_ page, you'll notice that The Lounge now supports push notifications (yes, even on mobile, except iOS), and that you can now access information about open sessions and even remotely log them out.
+
+<p align="center">
+  <img width="477" alt="Session list with the ability to disconnect remote sessions" src="https://user-images.githubusercontent.com/113730/31530598-a6c86b46-afaf-11e7-9272-eb742d328686.png">
+  <br>
+  <em>Current and remote sessions</em>
+</p>
+
+As a server administrator, you might be interested by a few new additions: themes can now be [retrieved from npm](https://www.npmjs.com/search?q=keywords%3Athelounge-theme) and installed using a new CLI command `lounge install <theme-name>`; integration with LDAP has been completely refactored; The Lounge can now be bound to Unix sockets.
```

Comment:
As a server administrator, you might be interested by a few new additions: 


- themes can now be retrieved from npm and installed using a new CLI command (`lounge install <theme-name>`)
- integration with LDAP has been completely refactored
- The Lounge can now be bound to Unix sockets

Reply:
in a few new additions*